runtime/browser/xwalk_content_browser_client.cc

```diff
@@ -93,22 +92,6 @@ namespace {
 // The application-wide singleton of ContentBrowserClient impl.
 XWalkContentBrowserClient* g_browser_client = nullptr;
 
-// A provider of services for Geolocation.
```

Comment:
Are you sure about this? It looks like you're removing the code without providing any replacement.